extensions/data-workspace/postInstall.js

```diff
@@ -0,0 +1,8 @@
+const fs = require('fs');
+
+// copy git.d.ts from git extension to data-workspace extension
+fs.copyFile('../git/src/api/git.d.ts', './src/typings/git.d.ts', (err) => {
```

Comment:
Why are you doing this? You shouldn't need to

Reply:
I tried directly referencing the git.d.ts in the git extension with `import * as git from '../../git/src/api/git';`, but was getting this error 
![image](https://user-images.githubusercontent.com/31145923/116592015-59ea6a00-a8d4-11eb-9fdf-2c56ceea0fc4.png)

I couldn't figure out why with the reference, main.js was being put in data-workspace/out/data-workspace/src/main.js 
![image](https://user-images.githubusercontent.com/31145923/116592132-7d151980-a8d4-11eb-90ac-381ba16c50a8.png)

but without the reference, it was in data-worksapce/out/main.js like expected:
![image](https://user-images.githubusercontent.com/31145923/116592182-8d2cf900-a8d4-11eb-9649-e88f4c2b5de9.png)

Reply:
I don't know how that happened - that should have nothing to do with the typings file....did you do a clean before rebuilding?

I opened a PR with what I did locally and that seems to work fine for me : https://github.com/microsoft/azuredatastudio/pull/15286

Pull it down and let me know if you're still having issues even with those changes.

Reply:
Ok that worked. I thought I was doing a clean before rebuilding yesterday, but maybe I forgot or my machine just needed to be restarted. At least I got to learn how postInstall works :)